bundles/org.openhab.core.io.console.karaf/src/main/java/org/openhab/core/io/console/karaf/internal/CommandWrapper.java

```diff
@@ -52,8 +52,8 @@ public class CommandWrapper implements Command, Action {
 
     /**
      * The constructor for the "help" instance of this class. This instance will be created by
-     * org.apache.karaf.shell.impl.action.command.ManagerImpl.instantiate(Class<? extends T>, Registry) and
-     * is used to print all usages from the `openhab` scope.
+     * {@code org.apache.karaf.shell.impl.action.command.ManagerImpl.instantiate(Class, Registry)} and
```

Comment:
Is the removal of the type parameter intentional?

Reply:
The removal didn't really improve anything so now it's back!